Apps/Laboratory/src/appsettings.hgdev.json

```diff
@@ -8,7 +8,7 @@
         "Authority": "https://dev.oidc.gov.bc.ca/auth/realms/ff09qn3f"
     },
     "Laboratory": {
-        "EndPoint": "https://phsahealthgatewayapi-dev.azurewebsites.net/api/v1/LabOrders"
```

Comment:
If you're changing existing config, we need to ensure that these values are not overridden in OpenShift.